devsetup/Makefile

```diff
@@ -76,36 +70,6 @@ edpm_compute_repos: ## Configure EDPM compute VM with repositories
 edpm_compute_cleanup: ## Delete EDPM compute VM
 	scripts/edpm-compute-cleanup.sh ${EDPM_COMPUTE_SUFFIX}
 
-.PHONY: edpm_play
-edpm_play: export EDPM_OVN_METADATA_AGENT_TRANSPORT_URL=$(shell oc get secret rabbitmq-transport-url-neutron-neutron-transport -o json | jq -r .data.transport_url | base64 -d)
-edpm_play: export EDPM_OVN_METADATA_AGENT_SB_CONNECTION=$(shell oc get ovndbcluster ovndbcluster-sb -o json | jq -r .status.dbAddress)
-edpm_play: export EDPM_OVN_DBS=$(shell oc get ovndbcluster ovndbcluster-sb -o json | jq -r '.status.networkAttachments."openstack/internalapi"[0]')
-edpm_play: export EDPM_OVN_METADATA_AGENT_NOVA_METADATA_HOST=127.0.0.1
-edpm_play: export EDPM_OVN_METADATA_AGENT_PROXY_SHARED_SECRET=12345678
-edpm_play: export EDPM_OVN_METADATA_AGENT_BIND_HOST=127.0.0.1
-edpm_play: export EDPM_NOVA_NOTIFICATIONS_TRANSPORT_URL=$(shell oc get secret rabbitmq-transport-url-nova-api-transport -o json | jq -r .data.transport_url | base64 -d)
-edpm_play: export EDPM_NOVA_COMPUTE_TRANSPORT_URL=$(shell oc get secret rabbitmq-transport-url-cell1-transport -o json | jq -r .data.transport_url | base64 -d)
-edpm_play: ## Deploy EDPM node using openstackansibleee resource
-	scripts/gen-ansibleee-ssh-key.sh
```

Comment:
We need to keep this script to generate secret.

Reply:
it is called when creating the nodes: https://github.com/openstack-k8s-operators/install_yamls/blob/master/devsetup/Makefile#L67-L68

Reply:
In Zuul CI side, nodepool provides the vm with public ip and pub key is available on the controller node.
We pass [SSH_KEY_FILE](https://github.com/rdo-infra/rdo-jobs/blob/master/playbooks/edpm/deploy_podified_control_plane.yaml#L171) with the edpm-play
```
 EDPM_NETWORK_CONFIG_TEMPLATE: "templates/ci/multinode.j2"
 SSH_KEY_FILE: "{{ ansible_user_dir }}/.ssh/id_rsa"
```
Above script creates the secrets for the node.

In CI workflow, we donot use ` edpm_compute` target. Can we add it back?